ui/pages/onboarding-flow/recovery-phrase/confirm-recovery-phrase.js

```diff
@@ -94,6 +97,7 @@ export default function ConfirmRecoveryPhrase({ secretRecoveryPhrase = '' }) {
           large
           className="recovery-phrase__footer__confirm--button"
           onClick={() => {
+            dispatch(setSeedPhraseBackedUp(true));
             history.push(ONBOARDING_COMPLETION_ROUTE);
           }}
```

Comment:
This action is async, so we should be awaiting it here. Otherwise the page navigation might complete before we finish setting this background state.

```suggestion
          onClick={async () => {
            await dispatch(setSeedPhraseBackedUp(true));
            history.push(ONBOARDING_COMPLETION_ROUTE);
          }}
```

The same goes for the other component this is called in as well.